TestCases/compliance-level-3/0084-feel-for-loops/0084-feel-for-loops.dmn

```diff
@@ -72,4 +72,38 @@
         </literalExpression>
     </decision>
 
+    <decision name="days in weekend" id="_daysInWeekend">
+        <variable name="days in weekend"/>
+        <literalExpression>
+            <text>["saturday", "sunday"]</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_014" id="_decision_014">
+        <variable name="decision_014"/>
+        <informationRequirement>
+            <requiredDecision href="#_daysInWeekend"/>
+        </informationRequirement>
+        <literalExpression>
+            <text>for i in days in weekend return if i = "sunday" then true else false</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="in a list" id="_inAList">
```

Comment:
from the DMN spec:

> A name start (grammar rule 26) SHALL NOT be a literal terminal symbol.

and again according to DMN spec FEEL grammar `in` is a literal terminal symbol.

This makes this model invalid.